tests/jvm/src/test/resources/metac-metacp.expect.diff

```diff
@@ -3,9 +3,8 @@ advanced.D#
 ===========
 --- metac
 +++ metacp
-                 type_arguments {
-                   tag: TYPE_REF
```

Comment:
Why is the `type_arguments` thingie gone?

Reply:
I think it moved out of the diff context due to `prefix {`